README.md

```diff
@@ -1 +1,152 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
```

Comment:
[OPTIONAL] Kindly remove unused codes from your project so as to make your code neater. 😊 📗